_posts/2018-08-02-Rust-1.28.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Announcing Rust 1.28"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.28.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.28.0 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.28.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1280-2018-08-02
+
+## What's in 1.28.0 stable
+
+### Global Allocators
+
+Allocators are the way that programs in Rust obtain memory from the system at
+runtime. Previously, Rust did not allow changing the way memory is obtained,
+which prevented some use cases. With 1.28.0, the `#[global_allocator]` attribute
+is now stable, which allows Rust programs to set their allocator to the system
+allocator, as well as define new allocators by implementing the [`GlobalAlloc`]
+trait.
+
+The standard library provides a handle to the system allocator, which can be
```

Comment:
Do you think it's worth mentioning jemalloc at some point? It took me a minute to realize this isn't the default behavior